ansible/roles/setup-kafka/tasks/main.yml

```diff
@@ -12,3 +12,11 @@
   when: kafka_id=="1" and item.retention_time is defined  
   tags:
     - processing-kafka
+
+- name: create topics
+  command: /opt/kafka/bin/kafka-topics.sh --zookeeper localhost:2181 --create --topic {{env_name}}.{{item.name}} --partitions {{ item.num_of_partitions }} --replication-factor {{ item.replication_factor }}
+  with_items: "{{ml_service_topics}}"
+  ignore_errors: true
+  when: when: kafka_id=="1" and ml_kafka_topic_create == true
```

Comment:
this should be a boolen check `ml_kafka_topic_create is true`
`==` should only be used for strings

Reply:
changed this variable when: kafka_id=="1" and ml_kafka_topic_create is true